extractor/src/main/java/org/schabi/newpipe/extractor/services/youtube/extractors/YoutubeMixPlaylistExtractor.java

```diff
@@ -67,8 +68,11 @@ public void onFetchPage(@Nonnull final Downloader downloader)
     @Nonnull
     @Override
     public String getName() throws ParsingException {
-        final String name = playlistData.getString("title");
+        String name = playlistData.getString("title");
         if (name == null) {
+            YoutubeParsingHelper.getTextFromObject(playlistData.getObject("titleText"));
```

Comment:
Use the `getTextAtKey` I introduced yesterday to solve the same issue but for Trending

Reply:
done